pkg/statsd/statsd_test.go

```diff
@@ -21,6 +21,9 @@ import (
 // TestStatsdThroughput emulates statsd work using fake network socket and null backend to
 // measure throughput.
 func TestStatsdThroughput(t *testing.T) {
+	if testing.Short() {
+		t.Skip()
```

Comment:
Would it be worthwhile adding a message to the skip so it is clear?

Reply:
You need to explicitly say `-short`, it's not skipped by default.  If you're doing it, you're doing it deliberately.

Reply:
Sounds good :)